docs/getting-started.md

```diff
@@ -67,6 +67,9 @@ kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/bu
 If you want to add Windows Nodes to your cluster, please refer to the
 installation instructions in [windows.md](/docs/windows.md).
 
+If you want to deploy Antrea on a cluster with IPv6 Service CIDR, please fill in the `ServiceCIDRv6`
```

Comment:
could you wrap the line into 80 chars as other lines do?

Reply:
Updated.